libraries/AP_BoardConfig/AP_BoardConfig_CAN.cpp

```diff
@@ -118,7 +118,7 @@ void AP_BoardConfig_CAN::setup_canbus(void)
             if (hal.can_mgr[drv_num - 1] != nullptr) {
                 initret &= hal.can_mgr[drv_num - 1]->begin(_var_info_can[i]._can_bitrate, i);
             } else {
-                printf("Failed to initialize can interface %d\n\r", i + 1);
+                hal.console->printf("Failed to initialize can interface %d\n\r", i + 1);
```

Comment:
printf was there on purpose.  hal.console is not available on pixhawk when driver loads.
So I would advise putting ifdef to give different functionality to linux boards.

Reply:
ok, maybe it's easier to revert that change?
I'll put printf back in place as printf is also supported on all linux boards

Reply:
If that is OK on linux boards - then yes.

Reply:
revert pushed